zhaquirks/tuya/ts0601_gas.py

```diff
@@ -1,123 +1,55 @@
-"""Gas Sensor."""
+"""Tuya Gas Sensor."""
 
-import logging
-
-import zigpy.profiles.zha
-from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.quirks.v2 import EntityPlatform, EntityType
+from zigpy.quirks.v2.homeassistant import CONCENTRATION_PARTS_PER_MILLION
+from zigpy.quirks.v2.homeassistant.sensor import SensorDeviceClass, SensorStateClass
 import zigpy.types as t
-from zigpy.zcl.clusters.general import Basic, Groups, Ota, Scenes, Time
-from zigpy.zcl.clusters.security import IasZone
-
-from zhaquirks import Bus
-from zhaquirks.const import (
-    DEVICE_TYPE,
-    ENDPOINTS,
-    INPUT_CLUSTERS,
-    MODELS_INFO,
-    OUTPUT_CLUSTERS,
-    PROFILE_ID,
-    ZONE_STATUS,
-    ZONE_TYPE,
-)
-
-from . import TuyaManufCluster, TuyaManufClusterAttributes
-
-_LOGGER = logging.getLogger(__name__)
-
-TUYA_GAS_DETECTED_ATTR = 0x0401  # [0]/[1] [Detected]/[Clear]!
 
-
-class TuyaGasDetectorCluster(TuyaManufClusterAttributes):
-    """Manufacturer Specific Cluster of the TS0601 gas detector."""
-
-    attributes = TuyaManufClusterAttributes.attributes.copy()
-    attributes.update(
-        {
-            TUYA_GAS_DETECTED_ATTR: ("gas_detected", t.uint8_t, True),
-        }
+from zhaquirks.tuya import TuyaPowerConfigurationCluster2AA
+from zhaquirks.tuya.builder import TuyaQuirkBuilder
+
+
+class TuyaSelfTestResult(t.enum8):
+    """Tuya self test result enum."""
+
+    Checking = 0x00
+    Success = 0x01
+    Failure = 0x02
+    Others = 0x03
+
+
+(
+    TuyaQuirkBuilder("_TZE200_ggev5fsl", "TS0601")
+    .applies_to("_TZE200_hr0tdd47", "TS0601")
+    .applies_to("_TZE200_rjxqso4a", "TS0601")
+    .applies_to("_TZE284_rjxqso4a", "TS0601")
+    .tuya_gas(dp_id=1)
+    .tuya_sensor(
+        dp_id=2,
+        attribute_name="co",
+        type=t.int16s,
+        device_class=SensorDeviceClass.CO,
+        state_class=SensorStateClass.MEASUREMENT,
+        unit=CONCENTRATION_PARTS_PER_MILLION,
+        fallback_name="CO concetration",
```

Comment:
This doesn't have a `translation_key` (as expected), so we wouldn't see this, but should still fix it 😄 
```suggestion
        fallback_name="CO concentration",
```